### DISCUSSION TITLE
Redirect issues

### DISCUSSION THREAD

**Discussion by FiereEinar** [community/community#115466] [Mobile]
Please make it so that it doesn't redirect to the mobile app when using github on a mobile browser on each click. Thank you.

**Comment by kangarruu** [comment]
Hello, thanks for writing in and for being a part of the GitHub Mobile community! Would you mind elaborating a bit more about your experience? What browser are you using when this redirection  happens? Is this happening during the login process or while you’re already logged into your account in the browser? Are there specific actions that consistently lead to redirection? If you could provide us with some more details it would help us troubleshoot. Thanks in advance!

**Reply by FiereEinar** [reply]
I'm sorry if my comment was very vague. I'm using google chrome. It happens during the login process and also when you're already logged in. Each time you click on any links like Your profile, Your repositories, the repos itself, or I think anything that sends you to another page will redirect to the mobile app. I just long press the links and open it in new tab to get around this.

There are cases where I need to use the chrome version because there are stuff that I wanna do that are not available in mobile app such as copying http/ssh link.

Thank you!

**Reply by kangarruu** [reply]
@FiereEinar no worries, thank you for the updated details! Perhaps turning off the system settings for opening supported links would work for you in this case? If you go to phone settings ->  Opening links -> GitHub -> and deselect 'Open supported links' that should prevent this redirection. Though I should note that this will also prevent direct github links from opening in the app by default. Please let us know if you experience any other unexpected issues with this setting turned off. I hope that helps!
